core/model/src/main/java/org/eclipse/sw360/antenna/model/license/License.java

```diff
@@ -0,0 +1,137 @@
+/*
+ * Copyright (c) Bosch.IO GmbH 2020.
+ *
+ * All rights reserved. This program and the accompanying materials
+ * are made available under the terms of the Eclipse Public License v2.0
+ * which accompanies this distribution, and is available at
+ * http://www.eclipse.org/legal/epl-v20.html
+ *
+ * SPDX-License-Identifier: EPL-2.0
+ */
+package org.eclipse.sw360.antenna.model.license;
+
+import java.util.*;
+
+public class License extends LicenseInformation {
+    private static final String THREAT_GROUP_KEY = "threatGroup";
+    private static final String CLASSIFICATION_KEY = "classification";
+
+    private String name;
```

Comment:
I would prefer to have id and name instead of name and longName. What is now the name is typically the SPDX identifiert or something similar and it is an id, so the code should reflect that.

Reply:
I know that, but I wanted to keep the old naming, but if it's okay to change the field/method names, I will do it.

What about the Configuration syntax? Should i keep the old:
```
<license>
   <name/>
   <longName/>
</license>
```
or should i change it to:
```
<license>
   <id/>
   </name/>
</license>
```

Reply:
That is more or less an internal naming, so I would prefer to have it changed to the second proposal. Could we somehow handle it in a way that in the tinaConfig we allow both somehow, but this collides with the name field, right? Perhaps we could phrase it fullName, this way we have to disjunctive sets.    Or we keep longName, actually we could use id and longName this would be a minimal change and allow also name to be used in tina config to reference the id.

Reply:
I do not see this remark being addressed? Do I get something wrong.

Idea is to name the two properties "id" and "longName" and allow also in the config to map name -> id.